### PR TITLE
Separated wallpaper showcases by platform

### LINKED ISSUE #8: Enormous total size of images loading
All the images in the READEME are being downloaded and rendered all at once. Right now, project size is 210.14 MB. It's too much. Besides, by the project's size growing over time, the issue escalates.
basically, I suggest division of topics/sub-topics.

### PR DESCRIPTION
**Pull request by iArmanKarimi**
Fixes #8 

Dividing pics by platform is the least to do and could be considered the first step to dividing pics. Thereafter, they can be put into different categories.
Additional changes:
- larger wallpaper platform header names ("##" -> "#")
- Index for each page
p.s. the structure of indices is untouched. I leave the owner to judge what to do with "statics and contributors" and "contributing" indices.

### CODE REVIEW

File: pages/Desktop.md [hunk 1]
```diff
@@ -0,0 +1,505 @@
+# Desktop Wallpapers
```
Comment (iArmanKarimi) [inline pages/Desktop.md:1]: Enlarged header

File: pages/Desktop.md [hunk 2]
```diff
@@ -0,0 +1,505 @@
...
+
+## Index
+
+- [Desktop Wallpapers](https://github.com/D3Ext/aesthetic-wallpapers#desktop-wallpapers/pages/Desktop.md)
+- [Mobile Wallpapers](https://github.com/D3Ext/aesthetic-wallpapers#mobile-wallpapers/pages/Mobile.md)
+- [Live Wallpapers](https://github.com/D3Ext/aesthetic-wallpapers#live-wallpapers/pages/Live.md)
+- [Statics && Contributors](https://github.com/D3Ext/aesthetic-wallpapers#statistics--contributors)
+- [Contributing](https://github.com/D3Ext/aesthetic-wallpapers#contributing)
```
Comment (iArmanKarimi) [inline pages/Desktop.md:9]: Index for navigation

**Review by iArmanKarimi — COMMENTED** [review]
Comments which indicate additional changes are also true for Live and Mobile.

**Comment by D3Ext** [comment]
Ok I've checked all your changes and I think that is a really good choice split wallpapers in different categories and even more sub categories so I merge the PR, thank you a lot!

**Pull request merged**